.github/workflows/npm-release.yml

```diff
@@ -0,0 +1,22 @@
+name: NPM Release
+on:
+  release:
+    types: [published]
+jobs:
+  release:
+    runs-on: ubuntu-latest
+    steps:
+      - uses: actions/checkout@v3
+        with:
+          ref: master
+      - uses: actions/setup-node@v3
+        with:
+          node-version: lts/*
+        env:
+          NODE_AUTH_TOKEN: ${{ secrets.NPM_TOKEN }}
+      - name: Release to NPM
+        run: |
+          npm i
+          npm start
+          npm run server

```

Comment:
Is there a reason you're running the server here? That's generally used for serving doc and watching for changes. If your desire is to just optimize and build I think you'll be fine with just `npm start`

Reply:
Thanks Paul, yeah I'm not familiar with the build process and I wanted to make sure all the bases were covered. E.g. CC needs to build doc for our releases via `docs-json`. Changing now!